cfme/control/explorer/alerts.py

```diff
@@ -376,6 +377,9 @@ def create(self, description, severity=None, active=None, based_on=None, evaluat
         view = navigate_to(self, "Add")
         if driving_event is None and evaluate is None:
             driving_event = view.driving_event.all_options[1].text
+        if severity is None and current_version() >= "5.11.0.7":
```

Comment:
We actually want to steer clear of using `current_version`, because it depends on 'magic' appliance creation.

We have direct access to the appliance object linked to our `AlertCollection`, and should use it instead. `self.appliance.version`

Reply:
Done